client/web/src/components/externalServices/ExternalServiceEditingTemporaryAlert.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+
+import { Alert, H4, Code, Text } from '@sourcegraph/wildcard'
+
+export const ExternalServiceEditingTemporaryAlert: React.FunctionComponent<React.PropsWithChildren<{}>> = () => (
+    <Alert variant="warning">
+        <H4>Edits will be reset when restarting</H4>
+        <Text className="mb-0">
+            Environment variable <Code>EXTSVC_CONFIG_ALLOW_EDITS</Code> is set along with{' '}
+            <Code>EXTSVC_CONFIG_FILE</Code>. Every change made through the UI will be undone with the next restart and
+            code host connection configuration is reset to the contents of <Code>EXTSVC_CONFIG_FILE</Code>.
```

Comment:
```suggestion
            code host connection configuration is reset to the contents of <Code>env.EXTSVC_CONFIG_FILE</Code>.
```

maybe to make clearer where that's coming from?

Reply:
I think capital case alone makes sense enough. The `env.` would actually throw me off